robottelo/config/base.py

```diff
@@ -706,6 +706,8 @@ def __init__(self, *args, **kwargs):
         self.otp_user = None
         self.time_based_secret = None
         self.disabled_user_ipa = None
+        self.group_users = None
+        self.groups = None
```

Comment:
Please add these new values to `robottelo.properties.sample` as well, just for consistency.

Reply:
Done